src/main/target/ANYFCF7/target.c

```diff
@@ -27,19 +27,19 @@
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
     DEF_TIM(TIM12, CH1, PB14, TIM_USE_PWM | TIM_USE_PPM,  TIMER_INPUT_ENABLED,  0 ), // S1_IN
     DEF_TIM(TIM12, CH2, PB15, TIM_USE_PWM,                TIMER_INPUT_ENABLED,  0 ), // S2_IN
-    DEF_TIM(TIM8,  CH1, PC6,  TIM_USE_PWM,                TIMER_INPUT_ENABLED,  0 ), // S3_IN DMA2_ST2 DMA2_ST2
-    DEF_TIM(TIM8,  CH2, PC7,  TIM_USE_PWM,                TIMER_INPUT_ENABLED,  0 ), // S4_IN DMA2_ST3 DMA2_ST2
-    DEF_TIM(TIM8,  CH3, PC8,  TIM_USE_PWM,                TIMER_INPUT_ENABLED,  0 ), // S5_IN DMA2_ST4 DMA2_ST2
-    DEF_TIM(TIM8,  CH4, PC9,  TIM_USE_PWM,                TIMER_INPUT_ENABLED,  0 ), // S6_IN DMA2_ST7
+    DEF_TIM(TIM8,  CH1, PC6,  TIM_USE_PWM,                TIMER_INPUT_ENABLED,  2 ), // S3_IN DMA2_ST2 DMA2_ST2
```

Comment:
I feel uncomfortable using dmaopt for TIMx_UP channels for following reasons.

1.  It is hard to distinguish TIMx_UP channels from non-TIMx_UP channels by looking at an integer value.

2. There are at least one other driver that uses DMA in non-TIMx_UP mode; namely LED strip.

How about introducing `TIMER_OUTPUT_TIMUP` flag and separate table of TIMx_UP streams/channels?